### ISSUE TITLE
implement lua api for mods 

### ISSUE THREAD

**Issue by freddyka** [ItsDeidara/The-Binding-of-Isaac-Mod-Conversion-Kit#3]
Right now the tool is pretty much useless because you are only able to change minor things inside the game

being able to use Lua code just like in the pc version is the goal of this issue

to implement this, the API for modded code needs to be created so that the tool can implement this inside the game somehow

**Comment by Licuatronix** [comment]
yeah.... i dont think that is posible

**Comment by StonedModder** [comment]
Correct. It is not. I tried my hardest when I made v2 a few weeks back 

There is nothing API related in the eboot any longer. You would need to make a modding API from the ground up
